swapserverrpc/withdraw.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+// We can't change this to swapserverrpc, it would be a breaking change because
+// the package name is also contained in the HTTP URIs and old clients would
+// call the wrong endpoints. Luckily with the go_package option we can have
+// different golang and RPC package names to fix protobuf namespace conflicts.
+package looprpc;
+
+option go_package = "github.com/lightninglabs/loop/swapserverrpc";
+
+service WithdrawalServer {
+    // WithdrawDeposits allows to cooperatively sweep deposits that haven't
+    // timed out yet to the client's wallet. The server will generate the
+    // partial sigs for the client's selected deposits.
+    rpc WithdrawDeposits (ServerWithdrawRequest)
+        returns (ServerWithdrawResponse);
+}
+
+message ServerWithdrawRequest {
```

Comment:
My main q here (commented on the server side too) whether it'd be better to just send the psbt of the sweep and the nonces.

Reply:
As discussed we don't draw an advantage here by using PSBT since the withdrawal tx won't dynamically add any inputs during signing.
The current implementation is also aligned with the reservations flow, so it might become easier to review.